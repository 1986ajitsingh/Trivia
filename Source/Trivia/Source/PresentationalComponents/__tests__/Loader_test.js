import renderer from 'react-test-renderer';
import React from 'react';
import ModuleUnderTest from '../Loader';

describe('renders correctly', () => {
  /* eslint-disable no-undef */
  test('normal', () => {
    /* eslint-enable no-undef */
    // Arrange/ Act
    const tree = renderer.create(<ModuleUnderTest loading />).toJSON();

    // Assert
    expect(tree).toMatchSnapshot();
  });
});
