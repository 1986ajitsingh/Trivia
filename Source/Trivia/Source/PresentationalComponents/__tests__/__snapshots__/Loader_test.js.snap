// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly normal 1`] = `
<Modal
  animationType="none"
  hardwareAccelerated={false}
  transparent={true}
  visible={true}
>
  <View
    style={
      Object {
        "alignItems": "center",
        "backgroundColor": "#EE82EE40",
        "flex": 1,
        "flexDirection": "column",
        "justifyContent": "space-around",
      }
    }
  >
    <View
      style={
        Object {
          "alignItems": "center",
          "backgroundColor": "#4B0082",
          "borderRadius": 10,
          "display": "flex",
          "height": 100,
          "justifyContent": "space-around",
          "width": 100,
        }
      }
    >
      <ActivityIndicator
        animating={true}
        color="#999999"
        hidesWhenStopped={true}
        size="large"
      />
    </View>
  </View>
</Modal>
`;
